src/codec_aom.c

```diff
@@ -1097,6 +1106,9 @@ static avifResult aomCodecEncodeImage(avifCodec * codec,
         // AVIF specification, Section 2.2.1. "AV1 Item Configuration Property":
         //   The values of the fields in the AV1CodecConfigurationBox shall match those
         //   of the Sequence Header OBU in the AV1 Image Item Data.
+        aomImage.cp = (aom_color_primaries_t)image->colorPrimaries;
+        aomImage.tc = (aom_transfer_characteristics_t)image->transferCharacteristics;
+        aomImage.mc = (aom_matrix_coefficients_t)image->matrixCoefficients;
```

Comment:
These are not part of the `AV1CodecConfigurationBox`. I suggest moving these lines away from the above comment.